measure-android/measure/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
     <uses-permission android:name="android.permission.INTERNET" />
+
+    <application>
+        <provider
+            android:name=".MeasureInitProvider"
```

Comment:
@abhaysood What priority should this content provider have? Should we make it high priority and initialize it before any other content provider?